### PR TITLE
Added Traveling info

### PR DESCRIPTION
**Pull request by OmkarPathak**
Added following Info on the **Travelling to Pune** page:
pycon1

pycon2

pycon3

pycon4

### CODE REVIEW

**Comment by OmkarPathak** [comment]
@SaptakS @sayanchowdhury is anything to be added to this page?

**Comment by SaptakS** [comment]
@OmkarPathak add the distance from Mumbai as well I think.

File: cauth/migrations/0006_auto_20171103_0204.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (SaptakS) [inline cauth/migrations/0006_auto_20171103_0204.py]: Why do we need this migration? Please re-check this.

File: root/urls.py [hunk 2]
```diff
@@ -15,7 +15,9 @@
         url(r'^about/$', root.views.about,
             name="about"),
         url(r'^sponsors/$', root.views.sponsors,
-            name="sponsors")
+            name="sponsors"),
+        url(r'^traveling/$', root.views.traveling,
```
Comment (SaptakS) [inline root/urls.py]: Please fix the `travelling` typo

**Review by SaptakS — CHANGES_REQUESTED** [review]

**Comment by OmkarPathak** [comment]
Added Distance from Mumbai, fixed typos

**Review by SaptakS — APPROVED** [review]

**Pull request merged**